option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message RebelShareRequest
{
 required U64Type serial_id = 1;
 repeated U64Type friends = 2;
}
